CMakeLists.txt

```diff
@@ -72,6 +73,28 @@ if (NOT CMAKE_ARCHIVE_OUTPUT_DIRECTORY)
   set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/lib")
 endif ()
 
+set(LIBRARY_DESTINATION "${CMAKE_INSTALL_LIBDIR}")
+set(MODULE_DESTINATION "")
```

Comment:
Since the code in VTK uses the `if(NOT DEFINED ...)` [^1] test, we should either:
1. initialize this variable using `vtk_module_python_default_destination` if a specific value hasn't been already passed when configuring this project
2. or introduce a variable called `_vtk_module_wrap_python_additional_args` and append to the list only when `VTK_WRAP_PYTHON AND VTK_WHEEL_BUILD` is TRUE



[^1]: https://github.com/Kitware/VTK/blob/4b2bdd7dd1a06fa0f561681e121d1f4f55110c64/CMake/vtkModuleWrapPython.cmake#L643-L645